Makefile

```diff
@@ -28,14 +27,6 @@ local_test:  start_services wait_for_services setup integration_tests stop_servi
 
 live_test: start_services wait_for_services setup live_integration_tests stop_services
 
-start_services:
```

Comment:
I know these didn't work before, but it's probably worth removing all traces of these target calls from this file too?